### DISCUSSION TITLE
Recurring Unpublish and Publish

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/publishpress-future#657] [Feature Requests]
On the example; let me try to give you a bit more info.
Client has a restaurant website.
Client is using posts for daily specials.
Each day say two posts (specials) will post. Example: say every M-F 9am to 2pm one special will post. Then at 3pm to 9pm another special will post.
I have posts created for each day of these.
So say there is 14 total (i.e. 2 per day, for 7 days), instead of them creating 14 new posts to schedule publishing each week, looking to see if I could just set these to automatically publish and unpublish the same posts every week.

Original request: https://secure.helpscout.net/conversation/2482214776/218779?viewId=6932143

**Comment by andergmartins** [comment]
We are considering this for the new Action Workflows feature. 
We are not sticking it to the "publish" and "unpublish" actions, but for any action the user needs.

To implement this we need a few changes:

* Improve the Schedule action to support specifying the date and time the action is running as the date source so we could have nested Schedule actions. After the first scheduled action is executed, it can schedule the next one. But this leads to a limited number of executions, according to the number of Schedule actions.
* Add an action that can trigger workflows, including itself.
* Implement the Conditional advanced action, which allows to decide a different path in the workflow, according to the current post status.

Another option is to add support to a loop between nodes, where a child node can be connected as input of a parent node. But this leads to an infinite loop issue in the editor (probably in the input/output handling). The workflow -> workflow action is probably safer.
